instrumentation/netty/netty-4-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4/common/client/NettySslInstrumentationHandler.java

```diff
@@ -20,6 +22,9 @@ public final class NettySslInstrumentationHandler extends ChannelDuplexHandler {
   private static final Class<?> SSL_HANDSHAKE_COMPLETION_EVENT;
   private static final MethodHandle GET_CAUSE;
 
+  private static final VirtualField<ChannelHandler, ChannelHandler> instrumentationHandlerField =
```

Comment:
Perhaps here should be a comment that the same virtual field is used elsewhere. Currently it is not obvious what it is for because in this file this virtual field is only set to null.